Authentication_and_Authorization/hello-security/list-users.php

```diff
@@ -6,22 +6,38 @@
 <body>
 <a href="index.php"> Take me home, country roads ... </a> <br>
 <?php
+
+include "db-connection.php";
+
+function checkIfRoleIsAdmin($email){
+    
+    $conn = openCon();
+
+    $sql = "SELECT * from person where email = '" . $email . "';";
+    $resultSet = $conn->prepare($sql);
+    $resultSet->execute() or die("Failed to query from DB!");;
+    $firstrow = $resultSet->fetch(PDO::FETCH_ASSOC);
+    
+    $role = $firstrow['role'];
+    return strcmp($role, "admin");
```

Comment:
You could extract here "admin" into variable to make the code more readable.

Reply:
Yes, actually it is a good idea that we do not have this constant hardcoded but extracted/configured somewhere instead.